test/test_extractor.py

```diff
@@ -314,8 +308,8 @@ def extension_list(self) -> list[str]:
 
     @pytest.mark.asyncio
     @pytest.mark.skipif(
-        sys.version_info.major == 3 and (sys.version_info.minor in (7, 11)),
-        reason="py3.7 and py3.11 fail sometimes",
+        sys.version_info.major == 3 and sys.version_info.minor == 11,
+        reason="py3.11 fail sometimes",
```

Comment:
Same as another check:
```suggestion
        sys.version_info[:2] == (3, 11), reason="py3.11 fails sometimes"
```